### PR TITLE
Pass a handler to HTTP.ErrorLog.

### PR DESCRIPTION
**Pull request by jameinel**
Our test suite spits out a bunch of stiple on stderr because its
the default log path for HTTP errors. However, we'd really like those
in the log file anyway, and we certainly don't want them in test run
output.
This changes our API server to forward all HTTP Error messages at Warning
level because the stdlib log.Logger is a concrete struct rather than an
interface.

### CODE REVIEW

File: apiserver/apiserver.go [hunk 1]
```diff
@@ -279,6 +280,21 @@ func (srv *Server) Wait() error {
...
+// interface, so we can't just proxy all of the log levels without inspecting
+// the string content. For now, we just want to get the messages into the log
+// file.
+type loggoWrapper struct {
+	logger loggo.Logger
+	level  loggo.Level
+}
+
+func (w *loggoWrapper) Write(content []byte) (int, error) {
+	w.logger.Logf(w.level, "%s", string(content))
```
Comment (howbazaar) [inline apiserver/apiserver.go]: The line number and file will be wrong, but perhaps good first cut.

**Comment by jameinel** [comment]
$$merge$$

**Comment by jameinel** [comment]
$$merge$$

**Comment by jameinel** [comment]
$$merge$$

**Pull request merged**